kafka-ui-api/src/main/java/com/provectus/kafka/ui/controller/SchemasController.java

```diff
@@ -60,7 +60,8 @@ protected KafkaCluster getCluster(String clusterName) {
   @Override
   public Mono<ResponseEntity<Void>> deleteSchema(
       String clusterName, String subjectName, ServerWebExchange exchange) {
-    return schemaRegistryService.deleteSchemaSubjectEntirely(getCluster(clusterName), subjectName);
+    return schemaRegistryService.deleteSchemaSubjectEntirely(getCluster(clusterName), subjectName)
```

Comment:
@jonasgeiregat 
Mono<Void> cant be mapped, you should use thenReturn(..) or then() on Mono<Void>. 
In general I think its better to make all schemaRegistryService.deleteXXX methods return Mono<Void> and do .thenReturn(ResponseEntity.ok().build()); in SchemasController

Reply:
Thanks for pointing this out. I improved the PR according to your input.
I also believe the mapping to a ResponseEntity should be done in the controller and definitely not forward the ResponseEntity from a previous Rest call.